bigquery/table_integration_test.go

```diff
@@ -592,3 +592,70 @@ func TestIntegration_TableUseLegacySQL(t *testing.T) {
 		_ = view.Delete(ctx)
 	}
 }
+
+func TestIntegration_TableDefaultCollation(t *testing.T) {
+	// Test DefaultCollation for Table.Create and Table.Update
+	if client == nil {
+		t.Skip("Integration tests skipped")
+	}
+	ctx := context.Background()
+	table := dataset.Table(tableIDs.New())
+	err := table.Create(context.Background(), &TableMetadata{
+		Schema:           schema,
+		DefaultCollation: CaseInsensitiveCollation,
+		ExpirationTime:   testTableExpiration,
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer table.Delete(ctx)
+	md, err := table.Metadata(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if md.DefaultCollation != CaseInsensitiveCollation {
+		t.Fatalf("expected default collation to be `%v`, but found `%v`", CaseInsensitiveCollation, md.DefaultCollation)
```

Comment:
Given the type, %q is probably more appropriate for the formatter.

Reply:
that's a good one, I didn't know about `%q`